docs/userGuide/cliCommands.md

```diff
@@ -69,7 +69,7 @@ Deploys the site to the repo's Github pages by pushing everything in the generat
 
 **Format:** `markbind init [<OPTIONS>]`
 
-**Description:** Init a markbind website project.
+**Description:** Init a Markbind website project.
```

Comment:
The B needs to be capitalized as well, that is how we brand our project. (i.e. `MarkBind`, not `Markbind`)